phone-book/src/contact.js

```diff
@@ -0,0 +1,227 @@
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+    [
+        {
+            name: 'Ivan',
+            lastName: 'Ivanov',
+            email: 'ivan-ivanov@vkontakte.com'
+        }
+    ],
+];
+
+
+class Contact{
+    constructor() {
+        this.contactTemplate = `
+            <div class="contact-block">
+
+                ${this.contactHeaderComponent()}
+
+                <main class="container contact-list">
+                    <table class="table table-hover table-striped">
+
+                        ${this.contactHeaderListComponent()}
+
+                        ${this.contactListComponent()}
+
+                    </table>
+                </main>
+
+                ${this.contactFooterComponent()}
+            </div>
+        `;
+    }
+
+    contactHeaderComponent(){
+        return `
+            <header class="container">
+                <form class="form-inline search-form row">
+                    <div class="form-group">
+                        <label class="sr-only" for="search">Search</label>
+                        <input type="text" class="form-control" id= "search" placeholder="Search">
+                    </div>
+                </form>
+            </header>
+        `
+    }
+
+    contactHeaderListComponent() {
+        return `
+            <thead>
+                <tr>
+                    <th>Name</th>
+                    <th>Last name</th>
+                    <th>Email</th>
+                </tr>
+            </thead>
+        `
+    }
+
+    contactListComponent() {
+        const contactList = document.createElement('tbody');
+        contactList.classList.add('contact-list');
+
+        users.forEach(user => {
+            const userFirstName = user[0].name;
+            const userLastName = user[0].lastName;
+            const userEmail = user[0].email;
+
+            contactList.innerHTML +=  `
+                <tr class="user">
+                    <td> ${userFirstName} </td>
+                    <td> ${userLastName} </td>
+                    <td> ${userEmail} </td>
+                </tr>
+            `
+        })
+
+        return contactList.outerHTML;
+    }
+
+    contactFooterComponent() {
+        return `
+            <footer class="container">
+            
+                <div class="row navigation-panel">
+                
+                    <div class="col-4 d-flex justify-content-center">
+                        <a href="#">
```

Comment:
```html
 <a href="#">
                        <button class="navigation-button active">
                            <span class="far fa-address-book icon active"></span>
                        </button>
                        </a>
```
I guess we can make "component" which will be responsible of creating such links